index.rst

```diff
@@ -26,3 +26,10 @@ or with ``pip``::
    basic_examples/getting_started.ipynb
    basic_examples/reaction_datasets.ipynb
    basic_examples/torsiondrive_datasets.ipynb
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Cookbook
+
+
+   cookbook/cookbook.ipynb
```

Comment:
I imagined setting up examples like [Dask Examples](https://examples.dask.org). Would this structure be better?

Reply:
Sure. To clarify, would the current notebook be split into:

* Get molecules and geometries
* Dataset and ReactionDataset: list available methods, basis sets, programs, etc.
* Get all calculations done on a molecule

Reply:
Lets give it a go.